operator/controllers/seldondeployment_engine.go

```diff
@@ -286,13 +291,13 @@ func createEngineContainerSpec(mlDep *machinelearningv1.SeldonDeployment, p *mac
 			{ContainerPort: 8082, Name: "admin", Protocol: corev1.ProtocolTCP},
 			{ContainerPort: 9090, Name: "jmx", Protocol: corev1.ProtocolTCP},
 		},
-		ReadinessProbe: &corev1.Probe{Handler: corev1.Handler{HTTPGet: &corev1.HTTPGetAction{Port: intstr.FromString("admin"), Path: "/ready", Scheme: corev1.URISchemeHTTP}},
+		ReadinessProbe: &corev1.Probe{Handler: corev1.Handler{HTTPGet: &corev1.HTTPGetAction{Port: intstr.FromString("admin"), Path: "/ready", Scheme: corev1.URISchemeHTTPS}},
```

Comment:
Isn't this for the Java engine? Shouldn't it always be set to `corev1.URISchemeHTTP`?

Reply:
No, unfortunately it's carrying over legacy as that function also creates the operator - if you check the latest it actually is under an if statement to check https vs http, this one was for testing but seems to had made its way (but addressed in latest commit)